tests/Propel/Tests/Generator/Behavior/Validate/UniqueConstraintTest.php

```diff
@@ -29,6 +29,7 @@ public function testUniqueValidatorPass()
         $publisher = new Publisher();
         $publisher->setName('Happy Reading');
         $publisher->setWebsite('http://www.happyreading.com');
+        $publisher->save();
```

Comment:
Why does it need to be saved before validating?

Reply:
It was failing before saving, and other examples all saved, so I assumed so.

Reply:
After saving I wouldnt expect validation to do anything, so we might actually "cloak" the issue here this way.
It was working before, we should find out how it will work again now without the save() part.

Reply:
```
Propel\Tests\Generator\Behavior\Validate\UniqueConstraintTest::testUniqueValidatorPass
Failed asserting that false is true.
```

Without the save, we get the above. I don't know what's failing validation or how validation works.  Additionally, it fails on `master` branch as well.  So, I'm pretty sure the problem is unrelated to this PR.

Reply:
Interesting..